arangod/Cluster/RestClusterHandler.h

```diff
@@ -40,9 +40,6 @@ class RestClusterHandler : public arangodb::RestBaseHandler {
   /// _api/cluster/endpoints
   void handleCommandEndpoints();
 
-  /// _api/cluster/serverInfo
-  void handleCommandServerInfo();
-
   /// _api/cluster/agency-dump
   void handleAgencyDump();
```

Comment:
Isn't this a potentially blocking request (it depends on the agency to respond, doesn't it?).
If so, should it be executed in an I/O thread?